src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,415 @@
+	 */
+	public function __construct( ReaderThemes $reader_themes ) {
+		$this->reader_themes = $reader_themes;
+	}
+
+	/**
+	 * Gets the value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Key for the option.
+	 *
+	 * [--format=<format>]
+	 * : Get value in a particular format.
+	 * ---
+	 * default: var_export
+	 * options:
+	 *   - var_export
+	 *   - json
+	 *   - yaml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # Get option.
+	 * $ wp amp option get theme_support
+	 * standard
+	 *
+	 * # Get option in JSON format.
+	 * $ wp amp option get theme_support --format=json
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function get( $args, $assoc_args ) {
+		list( $option_name ) = $args;
+
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		$options = $this->get_options();
+
+		if ( $options instanceof WP_Error ) {
+			/* translators: %s: error message */
+			WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+		}
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Could not get "%s" option. Does it exist?', 'amp' ), $option_name ) );
+		}
+
+		WP_CLI::print_value( $options[ $option_name ], $assoc_args );
+	}
+
+	/**
+	 * Updates an option value.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to update.
+	 *
+	 * <value>
+	 * : The new value.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # Update plugin option.
+	 * $ wp amp option update theme_support reader
+	 * Success: Updated theme_support option.
+	 *
+	 * @alias set
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function update( $args, $assoc_args ) {
+		list( $option_name, $option_value ) = $args;
+
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		if ( ! in_array( $option_name, self::ALLOWED_OPTIONS, true ) ) {
+			/* translators: %1$s: option name, %2$s: list of allowed options */
+			WP_CLI::error( sprintf( __( 'The option "%1$s" is not among the following options that can currently be managed via CLI: %2$s', 'amp' ), $option_name, implode( ', ', self::ALLOWED_OPTIONS ) ) );
+		}
+
+		// Update type for some options.
+		if ( Option::SANDBOXING_LEVEL === $option_name ) {
+			$option_value = (int) $option_value;
+		}
```

Comment:
Why was this specific cast needed? Is it because it is a string and the following logic requires it to be an int:

https://github.com/ampproject/amp-wp/blob/c28b8f1403116e831e837667cc7fca88e6fae9c3/src/Sandboxing.php#L123-L129

It seems like the `Sandboxing` service should be doing that cast to int as well, since it is already doing so for bool:

https://github.com/ampproject/amp-wp/blob/c28b8f1403116e831e837667cc7fca88e6fae9c3/src/Sandboxing.php#L120-L122

Reply:
I believe when we update options using REST API then it first validates the options(and return any validation failures) through provided schema. In the case of `'0' and '1'` and `'true' and 'false'` with schema type as boolean, it runs it through the `rest_sanitize_boolean` first hence type cast is not required but in the case of enums it fails:
https://github.com/ampproject/amp-wp/blob/c28b8f1403116e831e837667cc7fca88e6fae9c3/src/Sandboxing.php#L56
https://github.com/ampproject/amp-wp/blob/c28b8f1403116e831e837667cc7fca88e6fae9c3/src/Sandboxing.php#L68-L72


In other words, it doesn't reach here:
https://github.com/ampproject/amp-wp/blob/c28b8f1403116e831e837667cc7fca88e6fae9c3/includes/options/class-amp-options-manager.php#L368 

> It seems like the Sandboxing service should be doing that cast to int as well, since it is already doing so for bool:

Type cast here is done by REST schema's `rest_sanitize_boolean` IMO.